### PR TITLE
feat(setup/productionize): s/scaling/productionize

### PR DESCRIPTION
**Pull request by lwander**
Also adds a note on log/metric config

### CODE REVIEW

**Comment by lwander** [comment]
Should alleviate some of the concerns with https://github.com/spinnaker/spinnaker.github.io/pull/817

File: setup/productionize/caching/externalize-redis.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ layout: single
 title:  "Externalize Redis"
 sidebar:
   nav: setup
+redirect_from: /setup/scaling/externalize-redis/
```
Comment (dorbin) [inline setup/productionize/caching/externalize-redis.md:6]: Do we need the redirect? This is all new content, right?

Comment (lwander) [reply setup/productionize/caching/externalize-redis.md:6]: This has already been linked to

File: setup/productionize/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+sidebar:
+  nav: setup
+---
+
+Spinnaker is a large system, made of many microservices, each intended to be
+scaled, restarted, and configured independently. This provides operators a
+great degree of flexibility, and allows Spinnaker to handle massive scale (1K+
+deployments/day, 10K+ managed machines). However, there is no one-size-fits-all
+approach for configuring Spinnaker; your organization's usage patterns will
+need to inform how to productionize your Spinnaker deployment.
```
Comment (dorbin) [inline setup/productionize/index.md]: I'm ok with "productionize" in the title and headings, but maybe in the first mention in text say "...infor how to prepare your Spinnaker deployment to be used in production."

**Review by dorbin — APPROVED** [review]
Couple of minor questions, but 👍

File: setup/productionize/index.md (hunk 2 above)
Comment (lwander) [reply setup/productionize/index.md]: SGTM

**Pull request merged**